src/view/com/modals/AddAppPasswords.tsx

```diff
@@ -136,6 +137,17 @@ export function Component({}: {}) {
     }
   }
 
+  const textInputRef = React.useCallback((node: TextInput | null) => {
+    if (!node) {
+      return
+    }
+
+    // `selectTextOnFocus` isn't working with `autoFocus={true}` or without a timeout going.
+    setTimeout(() => {
+      node.focus()
+    }, 0)
+  }, [])
```

Comment:
Appreciate what you were going for here, but it's a bit cludgey and didn't work in my testing. I went back to autofocus because that does work on iOS. I think web users will have to just suffer.